packages/mongo/package.js

```diff
@@ -80,6 +80,7 @@ Package.onUse(function (api) {
 });
 
 Package.onTest(function (api) {
+  api.use('npm-mongo', 'server');
```

Comment:
If you switch to using the `MongoInternals.NpmModules.mongodb.module` approach I mentioned earlier, this shouldn't be needed.